### PR TITLE
qos

### PR DESCRIPTION
**Pull request by henningonsbring**
## Description
QOS will be set to normal for production when running `cg deliver rsync`

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh qos-rsync`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Review by Mropat — APPROVED** [review]

File: cg/meta/sbatch.py [hunk 1]
```diff
@@ -12,3 +12,8 @@
 ERROR_RSYNC_FUNCTION = """
 echo "Rsync failed"
 """
+
+QOS = {
```
Comment (henrikstranneheim) [inline cg/meta/sbatch.py:16]: Could you use/add to the constants here: cg/constants/priority.py instead?

Comment (henningonsbring) [reply cg/meta/sbatch.py]: you mean like this? (see changes)

File: cg/constants/priority.py [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 PRIORITY_OPTIONS = list(PRIORITY_MAP.keys())
 
+RSYNC_ACCOUNT_TO_QOS = {
+    "production": "normal",
+    "development": "low",
```
Comment (henrikstranneheim) [inline cg/constants/priority.py:12]: And also attach it to the SlurmQos, something like:
```suggestion
    "development": SlurmQos.LOW,
```

File: cg/constants/priority.py [hunk 3]
```diff
@@ -12,3 +12,9 @@ class SlurmQos(StrEnum):
     LOW: str = "low"
     NORMAL: str = "normal"
     HIGH: str = "high"
+
+
+RSYNC_ACCOUNT_TO_QOS = {
```
Comment (henrikstranneheim) [inline cg/constants/priority.py:17]: Sorry for doing this review in a drive by fashion
```suggestion
SLURM_ACCOUNT_TO_QOS = {
```
Maybe the link between Slurm account and prio should be more general

**Review by Mropat — APPROVED** [review]

**Comment by Mropat** [comment]
@henningonsbring this pr :b: roken due to merge conflicts and kind of a big refactor in rsync_api. Can you re-do the format of error fucntion?

**Comment by henrikstranneheim** [comment]
@henningonsbring and @Mropat This seems done in #1234. Can we close this one?

**Comment by Mropat** [comment]
yes!

**Pull request closed without merging**